automation/scripts/generate-keys-and-ledger.sh

```diff
@@ -170,13 +176,29 @@ else
 
   build_keyset_from_testnet_keys $online_output_dir "online-fish"
   build_keyset_from_testnet_keys $offline_output_dir "offline-fish"
+
+  cp ${online_output_dir}/*libp2p*.peerid ${LIBP2PPEERS}
+
 fi
 echo "Online Fish Keyset:"
 cat keys/keysets/${TESTNET}_online-fish
 echo "Offline Fish Keyset:"
 cat keys/keysets/${TESTNET}_offline-fish
 echo
 
+
+if [[ -s "keys/libp2p/${TESTNET}/whale-block-producer-1" ]]; then
```

Comment:
Hmmm not sure I get this check...

Reply:
This is something similar we've done in other parts of the script as a quick way to (hopefully check) if we've already done work so we dont overwrite anything